chronos/asyncfutures2.nim

```diff
@@ -413,64 +415,76 @@ proc `or`*[T, Y](fut1: Future[T], fut2: Future[Y]): Future[void] =
   fut2.callback = cb
   return retFuture
 
-# ZAH: The return type here could be a tuple
-# This will enable waiting a heterogenous collection of futures.
+# proc all*[T](futs: varargs[Future[T]]): Future[void] =
```

Comment:
generally, git keeps a copy of the old code already..

Reply:
Commented version is not for git, its for me and my memory. Because when in month i will not remember that there was also nice version, even if Git keep track on it.